tests/test-core/jqUnit/js/jqUnit-browser.js

```diff
@@ -134,8 +134,8 @@ var jqUnit = jqUnit || {};
                 node = node[0];
             }
             else if (node.length !== undefined) {
-                jqUnit.assertEquals("Expected number of nodes " + message, expected.length, node.length);
-                for (var i = 0; i < node.length; ++i) {
+                jqUnit.assertEquals("Expected number of nodes ", expected.length, node.length);
```

Comment:
Rather than remove the message entirely, i feel that it is useful for grouping the set of assertions performed here. Perhaps you could rewrite it as:

```javascript
jqUnit.assertEquals(message + ": Expected number of nodes: " + expected.length, expected.length, node.length);
```

Please also double check all of the uses of jqUnit.assertNode to ensure that changing this assertion doesn't break any of the existing uses. If it does, please rethink the change here or update those locations as needed.